tests/e2e/default/enketo/forms/training-cards-text-only.xml

```diff
@@ -0,0 +1,95 @@
+<?xml version="1.0"?>
+<h:html xmlns="http://www.w3.org/2002/xforms" xmlns:h="http://www.w3.org/1999/xhtml" xmlns:ev="http://www.w3.org/2001/xml-events" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:jr="http://openrosa.org/javarosa" xmlns:orx="http://openrosa.org/xforms">
+    <h:head>
+        <h:title>Text Only Training</h:title>
+        <model>
+            <itext>
+                <translation lang="en">
+                    <text id="/training_cards_text_only/action_icons/action_icons_aligment/action_icons_image_1:label">
```

Comment:
If I upload this form with `cht upload-training-forms`, the resulting document `internalId` is `training_cards_text_only`, without the `training` prefix. Do users need to edit the form document manually after upload?

Reply:
There are some ways to define the `internalId` without _manually_ modifying the doc in CouchDB directly: 
- If they are using XLS forms, then in the `settings` sheet, they can define the `internalId` in the `form_id` column (this is the one we explained in the [technical guide to upload trainings](https://docs.google.com/document/d/12_UIYLUtaE-Iyl_GvjbnapQYhjS0oJgL/edit#heading=h.ql5e8kip3uhb) - will be available in cht-docs) 
- Or they can add the `internalId` in the `properties.json` file of the form, and when uploading the form, that will update the doc in CouchDB. Example: 
```
// my_first_onboarding.properties.json
{
  "title": "",
  "internalId": "training:my_first_onboarding",
  "context": {
    "start_date": "2023-02-22",
    "duration": 60,
    "user_roles": [ "chw" ]
  }
}
```

However, this is something we need to fix in cht-conf for contact forms and training forms; we have a ticket for that: https://github.com/medic/cht-conf/issues/537

Reply:
Thanks for explaining.